icu_benchmarks/data/split_process_data.py

```
@@ -1,48 +1,58 @@
-    file_names: dict[str] = gin.REQUIRED,
-    preprocessor: Preprocessor = PolarsClassificationPreprocessor,
+    file_names: dict[str, str] | Any = gin.REQUIRED,
+    preprocessor: type[
+        PolarsClassificationPreprocessor | PolarsRegressionPreprocessor
+    ] = PolarsClassificationPreprocessor,
     use_static: bool = True,
-    vars: dict[str] = gin.REQUIRED,
-    modality_mapping: dict[str] = {},
-    selected_modalities: list[str] = "all",
+    vars: dict[str, str | list[str]] | Any = gin.REQUIRED,
+    modality_mapping: dict[str, list[str]] = {},
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)